examples/websocket-client/websocket_client.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how WebSocket client can be used to connect to and interact with a WebSocket server using Ballerina.
```

Comment:
Edit: This BBE demonstrates how a WebSocket client can be used to connect to and interact with a WebSocket server using Ballerina.